composer.json

```diff
@@ -70,26 +70,27 @@
         "phpstan/phpstan-phpunit": "^1.1",
         "phpstan/phpstan-webmozart-assert": "^1.2",
         "phpunit/phpunit": "^9.5",
+        "rector/rector": "^0.18.2",
         "sylius-labs/coding-standard": "^4.0",
-        "sylius/grid-bundle": "^1.7 || v1.12.0-ALPHA.1",
-        "symfony/console": "^5.4 || ^6.4",
-        "symfony/dependency-injection": "^5.4 || ^6.4",
-        "symfony/dotenv": "^5.4 || ^6.4",
-        "symfony/stopwatch": "^5.4 || ^6.4",
-        "symfony/uid": "^5.4 || ^6.4",
-        "symfony/workflow": "^5.4 || ^6.4",
+        "sylius/grid-bundle": "^1.7 || dev-symfony-7",
+        "symfony/console": "^5.4 || ^6.4 || ^7.0",
+        "symfony/dependency-injection": "^5.4 || ^6.4 || ^7.0",
+        "symfony/dotenv": "^5.4 || ^6.4 || ^7.0",
+        "symfony/http-kernel": "^5.4 || ^6.4 || ^7.0",
+        "symfony/stopwatch": "^5.4 || ^6.4 || ^7.0",
+        "symfony/uid": "^5.4 || ^6.4 || ^7.0",
+        "symfony/workflow": "^5.4 || ^6.4 || ^7.0",
+        "symfony/messenger": "^5.4 || ^6.4 || ^7.0",
+        "symfony/serializer": "^5.4 || ^6.4 || ^7.0",
+        "symfony/security-bundle": "^5.4 || ^6.4 || ^7.0",
         "twig/twig": "^2.12 || ^3.0",
         "vimeo/psalm": "^5.20",
-        "rector/rector": "^0.18.2",
-        "symfony/messenger": "^5.4 || ^6.4",
-        "symfony/serializer": "^5.4 || ^6.4",
-        "symfony/security-bundle": "^5.4 || ^6.4",
         "willdurand/hateoas-bundle": "^2.0"
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<3.0",
         "jms/serializer-bundle": "<3.5",
-        "willdurand/hateoas-bundle": "<2.0"
+        "willdurand/hateoas-bundle": "<2.0 || ^2.6"
```

Comment:
It would be useful to add a note about the reason of this conflict to `CONFLICTS.md` file

Reply:
done.